### ISSUE TITLE
C# inferenceSession.Run Execution will be crash 

### ISSUE THREAD

**Issue by cvixxt** [microsoft/onnxruntime#18910]
### Describe the issue

Hello everyone, I have a question to ask.
When I execute inferenceSession.Run(input), my program will become unresponsive and crash, but I can't find the relevant error code to help me overcome this problem. As shown in the figure below, my program disappears directly after execution and does not return any error code.
But I used the OnnxRunTime CPU mode, it's can work, but the GPU mode can not to work.
Please help me.

![image](https://github.com/microsoft/onnxruntime/assets/52907433/24534b8e-bd31-4dc2-a8c4-f61872726d8b)

![image](https://github.com/microsoft/onnxruntime/assets/52907433/54f96bf7-4b3c-4cc0-9492-62df74c371d0)

My environment is

cuda 11.8
cuDnn 8.5.0.96
OnnxRunTime.GPU 1.16.0
frameworkNET6.0

### To reproduce

![image](https://github.com/microsoft/onnxruntime/assets/52907433/06023c23-f151-49d3-991a-4c0c1e4aefe4)

### Urgency

This is a bit anxious.

### Platform

Windows

### OS Version

11

### ONNX Runtime Installation

Built from Source

### ONNX Runtime Version or Commit ID

1.16.0 with GPU

### ONNX Runtime API

C#

### Architecture

X64

### Execution Provider

CUDA

### Execution Provider Library Version

CUDA 11.6  cuDnn 8.5.0.96

**Comment by skottmckay** [comment]
You could set the log severity level to ORT_LOGGING_LEVEL_VERBOSE using the SessionOptions class. Pass the sessino options to the InferenceSession constructor.

https://onnxruntime.ai/docs/api/csharp/api/Microsoft.ML.OnnxRuntime.SessionOptions.html

Not clear what you mean by `my program disappears directly after execution and does not return any error code.`. Is 'execution' in that statement the call to InferenceSession.Run or something more? 

Have you looked at the system logs for any error info using EventViewer?

Not clear how ExtractPixels is implemented and whether there are any potential issues with the lifetime of the input data. I'd try running with dummy data to rule out any issues like that. e.g. something simple like a local variable of a DenseTensor with random data just to test if it crashes or not.

**Comment by cvixxt** [comment]
Hi [skottmckay](https://github.com/skottmckay)

I've resolved the issue and, as mentioned in issue #18051, Onnxruntime 1.15.1 does support 4080.

I tested the following:

GeForce RTX 4090
CUDA 11.8
cudnn 8.9.4.25
TensorRT-8.6.1.6.Windows10.x86_64.cuda-11
onnxruntime-GPU 1.15.1

This method works for me. After I reimported cuDnn 8.9.4.25, both onnxruntime-gpu 1.14.1 and 1.15.1 worked. Thank you for your reply and answers. I'll end this discussion.
